route/build/src/network_topo.f90

```diff
@@ -717,6 +718,148 @@ SUBROUTINE REACH_LIST(&
  IMPLICIT NONE
  ! input variables
  INTEGER(I4B)      , INTENT(IN)                 :: NRCH            ! number of stream segments
+ logical(lgt)      , intent(in)                 :: doReachList     ! flag to compute the list of upstream reaches
+ type(var_ilength) , intent(inout), allocatable :: structNTOPO(:)  ! network topology structure
+ ! output variables
+ type(var_dlength) , intent(inout)              :: structSEG(:)    ! network topology structure
+ integer(i4b)      , intent(out)                :: NTOTAL          ! total number of upstream reaches for all reaches
+ integer(i4b)      , intent(out)                :: ierr            ! error code
+ character(*)      , intent(out)                :: message         ! error message
+ ! local variables
+ INTEGER(I4B)                                   :: IRCH,JRCH,KRCH  ! loop through reaches
+ logical(lgt)                                   :: processedReach(nRch)  ! flag to define if reaches are processed
```

Comment:
you prefer perfect alignment on comment?

Reply:
No